client/src/App.css

```diff
@@ -1,3 +1,32 @@
-.App {
+/* .App {
   text-align: center;
 }
+.footer{
+  text-align: center;
+} */
+/* .videoContainer {
+  width: 500px;
+  height: auto;
+} */
+
+/* Footer Section */
+/* .footer {
+  background-color: #f9f9f9;
+  padding: 20px;
+  text-align: center;
+  font-size: 14px;
+  color: #777;
+} */
+
+/* Hero Section */
+
+/* .hero {
+  background-color: #f9f9f9;
+  padding: 20px;
+  text-align: center;
+}
+
+.hero-title {
+  font-size: 28px;
+  margin-bottom: 20px;
+} */
```

Comment:
Try and delete code you dont need anymore, it gets messy with it all left around 😄

Reply:
just thought i'd need them in the future lol